qcengine/programs/nwchem/harvester.py

```diff
@@ -855,9 +855,9 @@ def harvest(in_mol: Molecule, nwout: str, **outfiles) -> Tuple[PreservingDict, N
     # If present, align the gradients and hessian with the original molecular coordinates
     #  NWChem rotates the coordinates of the input molecule. `out_mol` contains the coordinates for the
     #  rotated molecule, which we can use to determine how to rotate the gradients/hessian
-    amol, data = out_mol.align(in_mol, atoms_map=False, mols_align=True, verbose=0)
-
+    amol, data = out_mol.align(in_mol, atoms_map=True, verbose=0)
```

Comment:
Why remove the `mols_align=True`? This alignment is between the schema mol orientation and nwc's internal standard orientation, so final rmsd should be truly zero.

Reply:
The problem here is that if we run NWChem with auto symmetry detection, it can perturb atomic positions enough to get an RMSD of greater than the error-checking cutoff in QCEl's alignment routine. 

I've got a test case that shows the effect of turning on and off autosym toggling on/off the error with `mols_align=True`

What's your take? My vote is to keep `mols_align=False` so that the code will work with NWChem's autosym. NWChem (at least v7) does not print out the molecular after rotation but before symmetrization, which is what we'd need to get the proper mill for rotating the gradients/hessian.

Reply:
aha, incomplete docstring in models/molecule.py . try setting `mols_align` to a relaxed float https://github.com/MolSSI/QCElemental/blob/master/qcelemental/molutil/align.py#L146-L151

Reply:
Done! Thanks for pointing that out. 

PS. I'm updating the qcel docs for `mols_align`